modules/juce_graphics/colour/juce_ColourGradient.h

```diff
@@ -200,21 +214,13 @@ class JUCE_API  ColourGradient  final
     //==============================================================================
     Point<float> point1, point2;
 
-    /** If true, the gradient should be filled circularly, centred around
-        point1, with point2 defining a point on the circumference.
-
-        If false and isConical is also false, the gradient is linear between the
-        two points.
-    */
-    bool isRadial;
-
-    /** If true, the gradient should be filled conically, centred around point1,
-        with point2 defining the start/end position.
+    /** Defines which of the available gradient types should be used. */
+    GradientType gradientType;

```

Comment:
Instead of initialising in the constructor, how about using default initialisation? (available since C++11)
```
GradientType gradientType = Linear;
```

Reply:
Done!